aquarius/events/metadata_updater.py

```diff
@@ -461,13 +461,14 @@ def _get_price_updates_from_fixed_rate_exchange(
                 "Found price not None, setting "
                 f"address={self.ex_contract.address} and type as empty string."
             )
-            price_dict.update({"address": self.ex_contract.address, "type": "exchange"})
+            price_dict.update({"exchange_id": exchange_id, "type": "exchange"})
+            price_dict.update({"address": "", "type": ""})
         else:
             logger.info(
                 "Found price=None, setting address and type as empty string."
             )  # noqa
             price_dict.update({"address": "", "type": ""})
```

Comment:
these two lines can be combined. The "type" key and its empty value are just repeated. Please add "exchange_id": "" to the original update instruction.